src/app/components/client/toolbar/AnnouncementDialog.tsx

```diff
@@ -463,6 +464,8 @@ export const LocalizedAnnouncementString = (
     bundleMonthlyPrice: `$${billingInfo.bundle.monthly}`,
     bundleYearlyPrice: `$${billingInfo.bundle.monthly * 12}`,
     regularYearlyPrice: `$${billingInfo.bundle.individual * 12}`,
+    emailAddressesCount: CONST_MAX_NUM_ADDRESSES_PLUS,
+    date: "Dec. 17", // deadline for viewing data broker results and in-progress removals
```

Comment:
Needed to pass these `ftl` vars in order to support the 3 announcements going out.